rocketpy/prints/rocket_prints.py

```diff
@@ -45,8 +45,12 @@ def inertia_details(self):
         print(
             "Rocket Mass: {:.3f} kg (With Propellant)".format(self.rocket.totalMass(0))
         )
-        print("Rocket Inertia I: {:.3f} kg*m2".format(self.rocket.inertiaI))
-        print("Rocket Inertia Z: {:.3f} kg*m2".format(self.rocket.inertiaZ))
+        print("Rocket Inertia 11: {:.3f} kg*m2".format(self.rocket.dry_I_11))
```

Comment:
1. "Rocket Inertia" is soemthing quite ambiguous here as the user will not know if it considers the motor (without propellant) or no. I suggest letting it more explicit.
2. In the future, we should create a matrix plot here, just to facilitate things.

Reply:
PR #387 will address this one